_posts/en/2019-10-21-schnorr-taproot-workshop.md

```diff
@@ -51,13 +52,13 @@ session.
 [![Introduction](/img/posts/taproot-workshop/introduction.png)](https://www.youtube.com/watch?v=1gRCVLgkyAE&list=PLPrDsP88ifOVTEJf_jQGunDUS05M9GdIC)
 {:.center}
 
-John Newbery introduces the workshop, giving a summary of why schorr and
+John Newbery introduces the workshop, giving a summary of why schnorr and
 taproot are useful technologies. He also explains why Bitcoin Optech created
-the workshop, and what the aims of the workshop are.
+the workshop and what the aims of the workshop are.
```

Comment:
Nit: I think this sentence could be improved further. Workshop is used redundantly and the sentence ends a bit inelegantly with "and what the aims of the workshop are."

Perhaps: "He then outlines the workshop's aims" (or goals).